ledfx/devices/osc.py

```diff
@@ -92,7 +92,7 @@ def flush(self, data):
         starting_addr = self._config["starting_addr"]
 
         # Convert data to rgb tuple
-        colors = [(int(r), int(g), int(b)) for r, g, b in data]
+        colors = data.astype(int).clip(0, 255)
```

Comment:
Don't need to clip at a device level, it is handled at a virtual level.

https://github.com/LedFx/LedFx/blob/ff45a40a603c67c16469570b9cb5c6479074a5b8/ledfx/virtuals.py#L694

Reply:
I had seen it being done like this in other devices...
Right now only in nanoleaf:

```
ledfx\devices\nanoleaf.py
     171:             self.set_offline()
     172:             return
     173: 
     174:     def flush(self, data):
     175:         for panel, col in zip(
     176:             self.config["pixel_layout"], data.astype(int).clip(0, 255)
```
`                                                                     ^^^`

Reply:
Thanks, we will remove from nanoleaf - must have slipped through

Reply:
Keep the type cast, as incoming can be float.

Dump the clip() and we should be good.